packages/webcert/src/store/user/userMiddleware.test.ts

```diff
@@ -42,4 +42,13 @@ describe('Test certificate middleware', () => {
       expect(fakeAxios.history.get.length).toBe(0)
     })
   })
+
+  describe('Handle set unit', () => {
```

Comment:
More tests should be added to middleware. For example to test the success-function after setting unit and to test the url for the api call

Reply:
Is it enough with the test in CareProviderModal.test?

Reply:
This comment still holds and now also applies for the statistics